network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -174,7 +188,13 @@ class Petition(CTA):
         default='Thank you for signing too!',
     )
 
-    override_translatable_fields = [
+    translatable_fields = CTA.translatable_fields + [
+        SynchronizedField('campaign_id'),
+        TranslatableField('comment_requirements'),
+        TranslatableField('checkbox_1'),
+        TranslatableField('checkbox_2'),
+        TranslatableField('share_link_text'),
```

Comment:
I’m hitting this when translating petition snippets, I think both `share_link` and `share_link_text` are not used anymore, and could be safely removed?
I assume these were used before ShareProgress?

<img width="913" alt="Capture d’écran 2021-03-11 à 14 57 59" src="https://user-images.githubusercontent.com/1294206/110798173-cef4d800-8271-11eb-8aff-8616b2f7d40f.png">

Reply:
Hmm yeah, editable=False suggests it's not being used - at least not in a way that editors would have access to. Ill push up a reomval. Thanks for finding this Theo!